### DISCUSSION TITLE
Changing the name of the tag "feature" -> "enhancement"

### DISCUSSION THREAD

**Discussion by kangz12345** [snu-quiqcl/qiwis#32] [Ideas]
Sometimes it is hard to say that something is a new 'feature'.
For example, just changing a convention to improve the performance or readability.

For such cases, the word "enhancement" may fit well.

Any opinions?

**Comment by kangz12345** [comment]
@BECATRUE @Ahrrri

**Comment by BECATRUE** [comment]
I agree! I thought `feature` tag was too abused.

**Reply by kangz12345** [reply]
I updated the tag name!
